asreview/webapp/src/ProjectComponents/HistoryComponents/LabeledRecordCard.js

```diff
@@ -183,121 +195,139 @@ const LabeledRecordCard = (props) => {
-                        ? "Add note"
-                        : "Save another note before adding"
-                      : "Add note after simulation is finished"
+                    !isSimulationProject()
+                      ? disableConvertPrior(value.prior)
+                        ? "Prior knowledge cannot be converted"
+                        : note.editing !== value.id
+                        ? value.included === 1
+                          ? "Convert to irrelevant"
+                          : "Convert to relevant"
+                        : "Save note before converting"
+                      : "Cannot be converted in simulation mode"
                   }
                 >
                   <span>
                     <IconButton
                       disabled={
-                        props.isSimulating || disableAddNoteButton(value.id)
+                        isSimulationProject() ||
+                        disableConvertPrior(value.prior) ||
+                        isLoading ||
+                        note.editing === value.id
                       }
-                      onClick={() => handleClickAddNote(value.id)}
+                      onClick={() => {
+                        handleClickLabelConvert(value);
+                      }}
                     >
-                      <NoteAddOutlined
-                        fontSize={!props.mobileScreen ? "medium" : "small"}
-                      />
+                      {value.included === 1 ? (
+                        <Favorite
+                          color="error"
+                          fontSize={!props.mobileScreen ? "medium" : "small"}
+                        />
+                      ) : (
+                        <FavoriteBorder
+                          fontSize={!props.mobileScreen ? "medium" : "small"}
+                        />
+                      )}
                     </IconButton>
                   </span>
                 </Tooltip>
-              )}
-              {!props.is_prior && value.id === note.editing && (
-                <Tooltip title="Remove note">
-                  <span>
-                    <IconButton
-                      disabled={isLoading}
-                      onClick={() => handleClickRemoveNote(value)}
-                    >
-                      <KeyboardArrowUp
-                        fontSize={!props.mobileScreen ? "medium" : "small"}
-                      />
-                    </IconButton>
-                  </span>
-                </Tooltip>
-              )}
-            </CardActions>
-            <RecordCardNote
-              isLoading={isLoading}
-              record={value}
-              mobileScreen={props.mobileScreen}
-              mutate={mutate}
-              note={note}
-              setNote={setNote}
-              is_prior={props.is_prior}
-            />
-          </Card>
-        ))}
+                {props.is_prior && (
+                  <Tooltip title="Remove">
```

Comment:
```suggestion
                  <Tooltip title="Remove from prior knowledge">
```

What about this. Might be confusing to users (delete completely).

Reply:
yes! we don't want to give the impression the record is deleted. Maybe add **label**?
 ```
Remove label from prior knowledge
```